bluebutton-data-model-codegen-annotations/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/annotations/RifLayoutsGenerator.java

```diff
@@ -30,6 +30,12 @@
 	 */
 	String beneficiaryHistorySheet();
 
+	/**
+	 * @return the name of the sheet in the Excel file that contains the RIF layout
+	 *         for beneficiary history data
+	 */
+	String beneficiaryHistoryNewSheet();
```

Comment:
```suggestion
	String beneficiaryHistoryTempSheet();
```

Can we go with this naming, instead? That way, we don't end up with the odd underscores elsewhere.